src/Diagnostics.Logger/ETWProvider/DiagnosticsETWProvider.cs

```diff
@@ -282,6 +282,17 @@ public void LogFullAscInsight(string RequestId, string Source, string Message, s
                 WebsiteHostName);
         }
 
+        public void LogDevOpsApiException(string ResourceUri, string Message, string Details, string DiagEnvironment = null, string DiagWebsiteHostName = null)
```

Comment:
I believe this would require the Event attribute & its properties to be given. You can refer to other logging methods to see how its constructed.